### ISSUE TITLE
Does not launch up the dolphin.

### ISSUE THREAD

**Issue by saicologic** [axsh/wakame-dolphin#6]
```
akame@localhost:~/work/wakame-dolphin(fix-0.0.4)$ ./ruby/bin/ruby ./bin/dolphin_server -c ./config/dolphin.conf
D, [2013-09-20T13:38:31.021311 #158680] DEBUG -- : Terminating 8 actors...
D, [2013-09-20T13:38:31.033503 #158680] DEBUG -- : Shutdown completed cleanly
/home/wakame/work/wakame-dolphin/vendor/bundle/ruby/2.0.0/gems/reel-0.3.0/lib/reel/request.rb:12:in `<class:Request>': uninitialized constant HTTP::METHODS (NameError)
    from /home/wakame/work/wakame-dolphin/vendor/bundle/ruby/2.0.0/gems/reel-0.3.0/lib/reel/request.rb:4:in `<module:Reel>'
    from /home/wakame/work/wakame-dolphin/vendor/bundle/ruby/2.0.0/gems/reel-0.3.0/lib/reel/request.rb:3:in `<top (required)>'
    from /home/wakame/work/wakame-dolphin/vendor/bundle/ruby/2.0.0/gems/reel-0.3.0/lib/reel.rb:13:in `require'
    from /home/wakame/work/wakame-dolphin/vendor/bundle/ruby/2.0.0/gems/reel-0.3.0/lib/reel.rb:13:in `<top (required)>'
    from /home/wakame/work/wakame-dolphin/lib/dolphin/request_handler.rb:3:in `require'
    from /home/wakame/work/wakame-dolphin/lib/dolphin/request_handler.rb:3:in `<top (required)>'
    from ./bin/dolphin_server:96:in `run'
    from ./bin/dolphin_server:102:in `<main>'
```

this issue

```
uninitialized constant HTTP::METHODS
```

**Comment by saicologic** [comment]
stable version

```
    http (0.4.0)
      certified
      http_parser.rb
```

my enviroment it's crashed

```
    http (0.5.0)
      http_parser.rb
```

http gems bump up 0.5.0 at September 14
http://rubygems.org/gems/http

**Comment by saicologic** [comment]
Fixed version 0.0.5
